src/html/shared/footer.html

```diff
@@ -0,0 +1,28 @@
+<footer class="c-footer">
+    <div class="c-footer--top">
+        <div class="o-wrapper">
+            <div class="o-layout">
+                <div class="o-layout__item">
+                    <br> {% include "./share.html" %}
+                    <br> {% include "./accordion.html" %}
+                </div>
+            </div>
+        </div>
+    </div>
+
+    <div class="c-footer--bottom">
+        <div class="o-wrapper">
+            <ul class="o-list-bare o-list-inline">
+                <li class="o-list-bare__item o-list-inline__item c-footer--bottom_item">
```

Comment:
I'm just guessing but I'm pretty sure we don't need both `o-list-bare` and `o-list-inline` for inline lists

Reply:
Yeap.